pkg/flowaggregator/exporter/s3.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/flowaggregator/options"
+	"antrea.io/antrea/pkg/flowaggregator/s3uploader"
+)
+
+type S3Exporter struct {
+	s3Input         *s3uploader.S3Input
+	s3ExportProcess *s3uploader.S3ExportProcess
+}
+
+func buildS3Input(opt *options.Options) s3uploader.S3Input {
+	return s3uploader.S3Input{
+		BucketName:        opt.Config.S3Uploader.BucketName,
+		BucketPrefix:      opt.Config.S3Uploader.BucketPrefix,
+		Region:            opt.Config.S3Uploader.Region,
+		RecordFormat:      opt.Config.S3Uploader.RecordFormat,
+		Compress:          opt.Config.S3Uploader.Compress,
+		MaxRecordsPerFile: opt.Config.S3Uploader.MaxRecordsPerFile,
```

Comment:
why not store /embed the `Config` in `S3Input`, instead of copying fields individually?